themes/bootstrap3/js/combined-search.js

```diff
@@ -4,10 +4,10 @@ VuFind.combinedSearch = (function CombinedSearch() {
   function init(container, url) {
     VuFind.loadHtml(container, url, '', function containerLoad(responseText) {
       if (responseText.length === 0) {
-        container.hide();
-        let parent = container.parent();
-        while (parent.hasClass('js-hide-if-empty')) {
-          parent.hide();
+        container.style.display = "none";
+        let parent = container.parentNode;
+        while (parent.classList.contains('js-hide-if-empty')) {
+          $(parent).hide();
```

Comment:
@padmasreegade, this seems to have reverted to using jQuery. Was that intentional? (In any case, I assume work is currently in progress, so I won't look again until you let me know you're ready for another review. :-) )